client/src/components/SearchForm.js

```diff
@@ -0,0 +1,16 @@
+const SearchForm = ({handleSubmit, onChange, city}) => {
+    return (  <form onSubmit={handleSubmit}>
+        <input
+          id="city-name"
+          type="text"
+          placeholder="Please enter the city name"
+          name="city"
+          onChange={onChange}
+          value={city}
+          required
+        />
+        <input type="submit" value="Submit" onClick={handleSubmit} />
```

Comment:
I don't think this `onClick` event is necessary because an `input` type of `submit` will trigger the form's `onSubmit` handler so `handleSubmit` is probably getting triggered twice here.